model/registry/gen-ai.yaml

```diff
@@ -14,6 +14,34 @@ groups:
               stability: experimental
               value: "openai"
               brief: 'OpenAI'
+            - id: microsoft
+              stability: experimental
+              value: "microsoft"
+              brief: 'Microsoft'
+            - id: huggingface
+              stability: experimental
+              value: "huggingface"
+              brief: 'Hugging Face'
+            - id: google
+              stability: experimental
+              value: "google"
+              brief: 'Google'
+            - id: meta
```

Comment:
The problem I see here is that Meta is not a provider. They don't have an SDK. You can host their models on replicate, on Azure, your own infra. How should the instrumentation infer that it should set this value to Meta? From the model name?

Reply:
Are we defining model providers as the infrastructure providers or SDK providers?

I think users should use Meta as the model provider when they are using their Llama series models.